space-dodge-game/screen-orientation-resize/main.js

```diff
@@ -1,6 +1,11 @@
 document.addEventListener('DOMContentLoaded', function () {
-  if (screen.lockOrientation) {
-    screen.lockOrientation('landscape');
+  // Make the application fullscreen to lock the screen
+  if (document.documentElement.webkitRequestFullScreen) {
```

Comment:
Even with this change, the screen-orientation-* samples are not working as expected.

Seems the `document.documentElement.webkitRequestFullScreen()` API takes effect under only user interaction like click event. See https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/webapi/tct-screenorientation-w3c-tests/screenorientation/support/fullscreen.js#L52

Any tips? @rakuco, @haoxli, @bspencer